cpp/tests/lists/sort_lists_tests.cpp

```diff
@@ -24,39 +24,21 @@
 template <typename T>
 using LCW = cudf::test::lists_column_wrapper<T, int32_t>;
 
-namespace cudf {
-namespace test {
-
-auto generate_sorted_lists(lists_column_view const& input,
-                           order column_order,
-                           null_order null_precedence)
+auto generate_sorted_lists(cudf::lists_column_view const& input,
```

Comment:
Should we be generally avoiding `namespace cudf::test` in our test code? It seems sensible to me, because the tests aren't really part of the "test library" `cudf::test` (utility classes and wrappers belong in the test library). I just want to make sure I'm understanding the reasoning for this change correctly, since there are a large number of test files that scope their tests in the `cudf::test` namespace like this one did.

Reply:
I do think we should generally avoid putting test code in `cudf::test` mainly because it implicitly places the code in `cudf` namespace as well. This makes it difficult to determine what parts are part of cudf and what parts are part of the test as well as what parts come from `cudf::test`. Perhaps we can open an issue to discuss it.

Reply:
I agree with @bdice. Test code should not be part of the `cudf::test` namespace but we should probably put the code in some namespace instead of orphan code so we can have better organization. Maybe something like `cudf::unit_test`? Yeah so this may needs more discussion.

Reply:
I agree no namespace is required. Along the same lines it could be a good idea to remove instances of `using namespace cudf;` and `using namespace cudf::test;` to help clarify what code is directly from cudf and what is from the test(s).

Reply:
Resolving this with reference to #11734